eland/operations.py

```diff
@@ -1225,36 +1225,56 @@ def describe(self, query_compiler: "QueryCompiler") -> pd.DataFrame:
         )
 
     def to_pandas(
```

Comment:
Instead of modifying `to_pandas()` what are your thoughts on modifying on `to_csv()` to chunk the dataframes with `self.search_yield_pandas_dataframes(...)` and then progressively appending them with `to_csv()`.

Reply:
Basically  we have to support `show_progress` and the chunkingpart. If we combine these two it becomes another `to_pandas` method. 

So duplication of code

Reply:
I mean only 4 lto 5 lines will be different between two methods. If i re implement it.

Reply:
My problem with the proposed implementation is it changes `to_pandas()` to do two completely different things based on the value of a parameter. Whenever you see this you likely actually want two different functions instead of one function filling two roles.

The above pain that you mention of "having to reimplement to_pandas" be hinting that we need a separate method (maybe `iterframes`?) that does everything inside of `to_pandas()` except it yields dataframes instead of stitching them together at the end. Then we can change `to_pandas()` and `to_csv()` to use the new `iterframes` method.

What do you think about this?

Reply:
This looks good. Ill implement. Never thought of that😶😅

Reply:
@V1NAY8 Thanks for working on this. It would really benefit my use case as well! Would you be willing to continue your work here? 

If you don't have the time right now, that's fine of course. In that case, I might build upon your changes and try to work in @sethmlarson's suggestions, if that's okay with you.